src/components/Pega_Extensions_BarCode/demo.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import type { StoryObj } from '@storybook/react';
+import PegaExtensionsBarCode from './index';
+
+export default {
+  title: 'Widgets/BarCode',
+  argTypes: {
+    heading: {
+      control: 'text'
+    },
+    format: {
+      options: ['CODE128', 'EAN13', 'upc', 'CODE39', 'ITF14', 'MSI', 'pharmacode', 'codabar'],
```

Comment:
this should come from your type prop - see other components like displayFormat for Compare table